components/Learn.js

```diff
@@ -1,7 +1,30 @@
 import React from "react";
+import { Row, Col } from "react-bootstrap";
+import Topic from "../components/Topic";
 
 const Learn = () => {
-	return <div>Learn</div>;
+	return (
+		<div className='font-lexend text-lg'>
+			{" "}
+			<div className='flex justify-center items-center font-lexend font-bold text-4xl mb-4'>
+				Learn
+			</div>
+			<Row>
+				<Col className='flex justify-around'>
```

Comment:
Apply the same styles for each column, for right now, I would just stick with `flex justify-center items-center`